src/libcrun/container.c

```diff
@@ -92,6 +96,69 @@ struct sync_socket_message_s
 
 typedef runtime_spec_schema_defs_hook hook;
 
+// linux hooks
+const char *hooks[] = {
+  "prestart",
+  "createRuntime",
+  "createContainer",
+  "startContainer",
+  "poststart",
+  "poststop"
+};
+
+// linux namespaces
+static const char *namespaces[] = {
+  "cgroup",
+  "ipc",
+  "mount",
+  "network",
+  "pid",
+  "user",
+  "uts"
+};
+
+static const char *actions[] = {
+  "SCMP_ACT_ALLOW",
+  "SCMP_ACT_ERRNO",
+  "SCMP_ACT_KILL",
+  "SCMP_ACT_KILL_PROCESS",
+  "SCMP_ACT_KILL_THREAD",
+  "SCMP_ACT_LOG",
+  "SCMP_ACT_NOTIFY",
+  "SCMP_ACT_TRACE",
+  "SCMP_ACT_TRAP"
+};
+
+const char *operators[] = {
+  "SCMP_CMP_NE",
+  "SCMP_CMP_LT",
+  "SCMP_CMP_LE",
+  "SCMP_CMP_EQ",
+  "SCMP_CMP_GE",
+  "SCMP_CMP_GT",
+  "SCMP_CMP_MASKED_EQ",
+};
+
+static const char *archs[] = {
```

Comment:
I am still not sure if we must list the arches.  Not only we risk to not list all the supported values, but we also risk that the dynamically linked version of libseccomp doesn't have support for what we are listing here.

What do you think?  Wouldn't it be safer for now to just drop the arches list from the JSON output?

Reply:
Yup, it makes sense. Does that sound good to you if we keep the reference of it in the code for the future but drop the entry from the final output for now?

Reply:
I think it is better to remove the code altogether, we can easily add it back if we ever need it in this form